src/qml/doc/icon-policy.md

```diff
@@ -0,0 +1,39 @@
+# Icon policy
+This document provides guidelines for contributing icons to Bitcoin Core.
+
+## Preparing Icons
+Both an icon source file, in Scalable Vector Graphics (SVG) format,
+and an optimized production file, in Portable Network Graphics (PNG) format,
+are required for each icon.
+
+#### SVG Source File
+SVGs are used as source files because they can scale while retaining image quality.
+They are not used in production due to limited application support.
+If different-sized production (PNG) icons are required,
+they can be generated from the associated SVG source file in any vector-based tool.
+
+#### PNG Production File
+PNGs are used in production due to wide application support, transparency support,
+and better image quality compared to competing file types such as JPEG.
+
+#### Optimizing Production Files (PNG)
+Production (PNG) files must be processed by the [optimize-png.py](https://github.com/bitcoin-core/bitcoin-maintainer-tools/blob/master/optimise-pngs.py) script before their inclusion in Bitcoin Core.
+PNG optimization removes various unnecessary color profiles, ancillary (alla),
```

Comment:
> ... ancillary (alla), ...

I cannot translate it into my native language ((

Reply:
https://en.wikipedia.org/wiki/Ancillary_data

Reply:
i've changed the text here from "ancillary (alla)" to "ancillary data"